src/components/Reports/ReportsBox/ReportsBox.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { InjectAppServices } from '../../../services/pure-di';
+import { FormattedMessage } from 'react-intl';
+import './ReportsBox.css';
+
+class ReportsBox extends React.Component {
+  constructor({ dependencies: { datahubClient } }) {
+    super();
+
+    /** @type { import('../../services/datahub-client').DatahubClient } */
+    this.datahubClient = datahubClient;
+
+    this.state = {
+      visits: 0,
+      dateFrom: null,
+      dateTo: new Date().toLocaleDateString(),
```

Comment:
What do you think about directly using dates in place of strings?

Reply:
Fixed